apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -217,6 +231,16 @@ export function TokenDetails({ coinType }: TokenDetailsProps) {
                                         accountAddress={activeAccountAddress}
                                     />
                                 ) : null}
+                                {accountHasIota && hasSupplyIncreaseVestingObjects ? (
```

Comment:
why does the account need iotas to show the hints?
During migration, some accounts will have all their funds in outputs, and some will have no funds
for example: https://explorer.rebased.iota.org/address/0x3da5294f00a66b6a4ef5a62e19b0bcf3b0a076cfa0dd10fc9dc4de5245cc511e?network=https%3A%2F%2Fapi.iota-rebased-alphanet.iota.cafe